### ISSUE TITLE
exampleRunMultiCrit: the trafo function is not used for nsga2 

### ISSUE THREAD

**Issue by berndbischl** [mlr-org/mlrMBO#92]
Must be tested and fixed!

**Comment by jakobbossek** [comment]
@danielhorn Is this already done? I think you wrote `exampleRunMultiCrit`

**Comment by ja-thomas** [comment]
can someone provide reproducible example, I'm not sure what the error is exactly.

**Comment by danielhorn** [comment]
I think this should be the example. Unfortunately, there is an error at the moment.

```
library(mlrMBO)
library(ggplot2)
library(smoof)

set.seed(1)
configureMlr(show.learner.output = FALSE)
pause = interactive()

obj.fun = makeZDT1Function(dimensions = 2L)

learner = makeLearner("regr.km", nugget.estim = FALSE, predict.type = "se")

ctrl = makeMBOControl(n.objectives = 2L, propose.points = 2L, trafo.y.fun = trafoLog())
ctrl = setMBOControlTermination(ctrl, iters = 5L)
ctrl = setMBOControlInfill(ctrl, crit = "dib",
  opt.focussearch.points = 10000L)
ctrl = setMBOControlMultiCrit(ctrl, parego.s = 100)

design = generateDesign(5L, getParamSet(obj.fun), fun = lhs::maximinLHS)

run = exampleRunMultiCrit(obj.fun, design = design, learner = learner, ctrl,
  points.per.dim = 50L, show.info = TRUE, nsga2.args = list())

plotExampleRun(run, pause = pause, gg.objects = list(theme_bw()))

```

**Comment by berndbischl** [comment]
@danielhorn 
what is the state here? pls check

**Comment by danielhorn** [comment]
Was fixed via #278, just forgot to close.
